cloud/amazon/ec2.py

```diff
@@ -212,7 +212,7 @@
     required: false
     default: 'present'
     aliases: []
-    choices: ['present', 'absent', 'running', 'stopped']
+    choices: ['present', 'absent', 'running', 'rebooted', 'stopped']
```

Comment:
I thought initially that the AWS documentation was clear on the distinction between reboot and restart, however I just realize they made our discussion worse: http://docs.aws.amazon.com/cli/latest/reference/ec2/reboot-instances.html

"If a Linux/Unix instance does not cleanly shut down within four minutes, Amazon EC2 performs a hard reboot."

I'm in favor of calling it 'restarted' and having people who want a hard stop / start do this via the method described above. I'll get the fix out on monday.